pkg/drivers/kic/oci/cli_runner.go

```diff
@@ -131,7 +131,10 @@ func runCmd(cmd *exec.Cmd, warnSlow ...bool) (*RunResult, error) {
 	if warn {
 		if elapsed > warnTime {
 			out.WarningT(`Executing "{{.command}}" took an unusually long time: {{.duration}}`, out.V{"command": rr.Command(), "duration": elapsed})
```

Comment:
I don't think this should show for sudo either, as it may include the amount of time it took for a user to type their password.

Reply:
I was **not** planning to ask the user for their password, for running podman (it was supposed to be set up as NOPASSWD, so it should be able to run interactively without any password)

Since we run multiple podman commands at the same time, it ends up a bit confusing. And the regular sudo setup times out after 15 minutes, so it will come back as a surprise later on.